src/main/java/it/gov/pagopa/payhub/activities/connector/organization/client/BrokerClient.java

```diff
@@ -0,0 +1,26 @@
+package it.gov.pagopa.payhub.activities.connector.organization.client;
+
+import it.gov.pagopa.payhub.activities.connector.organization.config.OrganizationApisHolder;
+import it.gov.pagopa.pu.organization.dto.generated.PagedModelBroker;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Lazy
+@Service
+public class BrokerClient {
+	private static final int DEFAULT_PAGE_NUMBER = 0;
+	private static final int DEFAULT_PAGE_SIZE = 20;
+	private static final List<String> DEFAULT_SORT = List.of("asc");
```

Comment:
We are not interested on Sort, let it to be null (don't use a constant to store the null)